test/message/error_aggregateTwoErrors.out

```diff
@@ -11,3 +11,4 @@ AggregateError: original
     at node:internal/main/run_main_module:*:* {
   code: 'ERR0'
 }
+    via Node.js *
```

Comment:
This looks rather confusing that it's indented after the closing bracket. Thus, I would not do that. It's not that important though.

Reply:
Agreed.
Updated the PR without indentation and "via" keyword.
See: https://github.com/nodejs/node/pull/38332#issuecomment-910089818